src/main/java/com/commercetools/sync/producttypes/utils/AttributeDefinitionUpdateActionUtils.java

```diff
@@ -20,20 +29,139 @@
+        @Nonnull final AttributeDefinitionDraft newAttributeDefinition) {
+
+        return buildUpdateAction(oldAttributeDefinition.getInputTip(), newAttributeDefinition.getInputTip(),
+            () -> SetInputTip.of(
+                oldAttributeDefinition.getName(),
+                newAttributeDefinition.getInputTip()
+            )
+        );
+    }
+
+    /**
+     * Compares the 'isSearchable' fields of an {@link AttributeDefinition} and an
+     * {@link AttributeDefinitionDraft} and returns an {@link UpdateAction}&lt;{@link ProductType}&gt; as a result in
+     * an {@link Optional}. If both the {@link AttributeDefinition} and the {@link AttributeDefinitionDraft} have the
+     * same 'isSearchable' field, then no update action is needed and hence an empty {@link Optional} is returned.
+     *
+     * @param oldAttributeDefinition the attribute definition which should be updated.
+     * @param newAttributeDefinition the attribute definition draft where we get the new 'isSearchable' field.
+     * @return A filled optional with the update action or an empty optional if the 'isSearchable' fields are identical.
+     */
+    @Nonnull
+    public static Optional<UpdateAction<ProductType>> buildChangeIsSearchableUpdateAction(
+        @Nonnull final AttributeDefinition oldAttributeDefinition,
+        @Nonnull final AttributeDefinitionDraft newAttributeDefinition) {
+
+        return buildUpdateAction(oldAttributeDefinition.isSearchable(), newAttributeDefinition.isSearchable(),
+            () -> ChangeIsSearchable.of(
+                oldAttributeDefinition.getName(),
+                newAttributeDefinition.isSearchable()
+            )
+        );
+    }
+
+    /**
+     * Compares the input hints of an {@link AttributeDefinition} and an {@link AttributeDefinitionDraft} and returns
+     * an {@link UpdateAction}&lt;{@link ProductType}&gt; as a result in an {@link Optional}. If both the
+     * {@link AttributeDefinition} and the {@link AttributeDefinitionDraft} have the same input hints, then no update
+     * action is needed and hence an empty {@link Optional} is returned.
+     *
+     * @param oldAttributeDefinition the attribute definition which should be updated.
+     * @param newAttributeDefinition the attribute definition draft where we get the new input hint.
+     * @return A filled optional with the update action or an empty optional if the input hints are identical.
+     */
+    @Nonnull
+    public static Optional<UpdateAction<ProductType>> buildChangeInputHintUpdateAction(
+        @Nonnull final AttributeDefinition oldAttributeDefinition,
+        @Nonnull final AttributeDefinitionDraft newAttributeDefinition) {
+
+        return buildUpdateAction(oldAttributeDefinition.getInputHint(), newAttributeDefinition.getInputHint(),
+            () -> ChangeInputHint.of(
+                oldAttributeDefinition.getName(),
+                newAttributeDefinition.getInputHint()
+            )
```

Comment:
This fits on one line too.

Reply:
If I put in one line I get an error from checkstyle that line length is greater than 120 characters

`[ant:checkstyle] [WARN] /Users/javierortiz/Projects/commercetools/platform/commercetools-sync-java/src/main/java/com/commercetools/sync/producttypes/utils/AttributeDefinitionUpdateActionUtils.java:161: Line is longer than 120 characters (found 129). [LineLength]`

Reply:
It shouldn't be longer if you have it like that: 
````java
return buildUpdateAction(oldAttributeDefinition.getInputHint(), newAttributeDefinition.getInputHint(), 
            () -> ChangeInputHint.of(oldAttributeDefinition.getName(), newAttributeDefinition.getInputHint()));
````